src/plugins/telemetryTable/components/table.vue

```diff
@@ -462,21 +473,34 @@ export default {
         calculateColumnWidths() {
             let columnWidths = {};
             let totalWidth = 0;
-            let sizingRowEl = this.sizingTable.children[0];
-            let sizingCells = Array.from(sizingRowEl.children);
             let headerKeys = Object.keys(this.headers);
+            let sizingTableChildren = Array.from(this.sizingTable.children);
 
             headerKeys.forEach((headerKey, headerIndex)=>{
-                let cell = sizingCells[headerIndex];
-                let columnWidth = cell.offsetWidth;
-                columnWidths[headerKey] = columnWidth;
-                totalWidth += columnWidth;
+                sizingTableChildren.forEach(sizingRowEl => {
+                    let sizingCells = Array.from(sizingRowEl.children),
+                        cell = sizingCells[headerIndex];
+
+                    if (columnWidths[headerKey]) {
+                        let currentWidth = columnWidths[headerKey],
+                            newWidth = cell.offsetWidth;
+
+                        if (newWidth > currentWidth) {
```

Comment:
What happens if the newWidth is shorter than the current width? Do we only want to track increasing widths?

Reply:
Simplified it, does not need this if block anymore